content/django-vs-flask.md

```diff
@@ -0,0 +1,67 @@
+# Django or Flask which one is should you choose your project?
+Django and Flask are among the most popular Python web frameworks.
+In this article we're going to review their main strengths and weaknesess to help you decide wich one is the better fit for your project.
+
+
+# Django
+
+## Approach
+Django opts for an all batteries included approach.
+This means that we have everything we need to build our project included into the framework
+from a powerful ORM to an authentication system or a templating engine.
+Django takes care of it all without the need to learn any extra libraries.
+
+It also has the concept of plugabble apps, that means that you can drop an app folder inside your project and get it's functionality right away
+with very little configuration.
+
+## Main features
+- Django has a very powerful ORM that exposes a lot of SQL to the python programming language.
+- Authentication and Authorization built in, without any extra configuration
+- A built in admin site
+- A built in mailing system.
+
+## Pitfalls
+- Tightly coupled with it's toolset, django it's sort of difficult to use in combination with other tools (replacing the form library for example)
+- The default scafolding, while not bad for small projects, does not scales without modification when trying to manage things such as multiple environments.
```

Comment:
Maybe you can add here: regardless this point, exists cookie cutter to help us to get a more maintainable structure

Reply:
Even tough useful, cookiecutter is not an official tool nor it's included in the docs.
That's why i didn't added it here.